File: pyg_manager_web/src/main/webapp/js/controller/indexController.js
app.controller("indexController",function ( $scope,indexService) {
    //从SpringSecurity中去获取当前登录用户。
    $scope.showName=function () {
        indexService.showName().success(function (response) {
            $scope.username = JSON.parse(response);
        })
    }
})

File: pyg_manager_web/src/main/java/com/pyg/manager/controller/IndexController.java
package com.pyg.manager.controller;

import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/index")
public class IndexController {

    @RequestMapping("/showName")
    public String showName(){
        //从SpringSecurity中去获取用户名
        String username = SecurityContextHolder.getContext().getAuthentication().getName();
        return username;
    }
}
